integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/IngressSteps.java

```diff
@@ -79,8 +83,9 @@ public void ingressOfBridgeIsNotAvailableWithinMinutes(String testBridgeName, in
                         IngressResource.optionsJsonEmptyEventResponse(context.getManagerToken(), endpoint)
                                 .then()
                                 .statusCode(Matchers.anyOf(Matchers.is(404), Matchers.is(503)));
-                    } catch (UnknownHostException | SSLException ignored) {
+                    } catch (UnknownHostException | SSLException e) {
                         // The DNS was properly deleted and the record expired.
+                        LOGGER.info("An exception '{}' was raised when trying to call the bridge ingress, considering the bridge as deleted.", e.getMessage());
```

Comment:
The usual logging is unfortunately not working properly, please use:
`context.getScenario()..log(.....);`

https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/a1df45011f0475d1dd0cf19322f5e86c892cdcfb/integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/context/TestContext.java#L51